### ISSUE TITLE
FBSDK does not allow me to sign in different account after logout

### ISSUE THREAD

**Issue by KrisLau** [facebook/facebook-android-sdk#1075]
### Checklist before submitting a bug report

- [X] I've updated to the latest released version of the SDK
- [X] I've searched for existing [Github issues](https://github.com/facebook/facebook-android-sdk/issues)
- [X] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [X] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [X] This issue is not security related and can safely be disclosed publicly on GitHub

### Java version

8.0.2010.9

### Android version

Android 11

### Android SDK version

13.2.0

### Installation platform & version

Yarn

### Package

Login

### Goals

Login > logout > log in with different account

### Expected results

Should show option to log out and change account 

### Actual results

Log in screen assumes I'm using the same account with no options to change account or log out

### Steps to reproduce

I use [react-native-fbsdk-next](https://github.com/thebergamo/react-native-fbsdk-next) to use this SDK and there is already an issue there for this: 

- https://github.com/thebergamo/react-native-fbsdk-next/issues/29

The main maintainer there has stated that they cannot do anything unless the underlying Facebook SDK changes. There is also a demand for this as there was also a previous issue on the old deprecated https://github.com/facebookarchive/react-native-fbsdk/issues/712

### Code samples & details

```java
// INSERT YOUR CODE HERE
```

**Comment by yurika-e** [comment]
This is expected behavior that we do not currently support "switch account" on the FB Login dialog. The user can go to Facebook app and / or FB on browser to switch accounts.

**Comment by KrisLau** [comment]
@yurika-e I've tried changing the account on the browser and it does not seem to work. Even when I log out and don't log in to any account, it will just automatically log me back in to the same account. Reference this thread: https://github.com/thebergamo/react-native-fbsdk-next/issues/29

**Comment by KrisLau** [comment]
@yurika-e

**Comment by KrisLau** [comment]
@yurika-e

**Comment by KrisLau** [comment]
@dbacinski Yeah that's the same for me but it doesn't seem like facebook is planning to implement any fixes for this. I'm not sure how other apps implement it since I've definitely seen it in other apps. I'm assuming it has something to do with using an incognito mode webview somehow but im not sure
